### ISSUE TITLE
`no-empty-description` incorrectly flags non-empty descriptions.

### ISSUE THREAD

**Issue by neverendingqs** [lo1tuma/eslint-plugin-mocha#313]
The following seem like reasonable test descriptions, but they get flagged by `no-empty-description`:

```js
describe('eslint-plugin-mocha', function() {
  const description = 'description';
  it(description, function() {});

  ['test1', 'test2'].forEach(testname => {
    it(testname, function() {});
  });

  const removePrefix = 'testcase.prefix';
  it(removePrefix.replace('.prefix'), function() {});
});
```

```
   3:3  error  Unexpected empty test description  mocha/no-empty-description
   6:5  error  Unexpected empty test description  mocha/no-empty-description
  10:3  error  Unexpected empty test description  mocha/no-empty-description
```

Was this intended with #304?

**Comment by lo1tuma** [comment]
Thanks for reporting. This is definitely a bug.

**Comment by neverendingqs** [comment]
I still get the following error with [`eslint-plugin-mocha@10.0.2`](https://github.com/lo1tuma/eslint-plugin-mocha/releases/tag/10.0.2):

```
  10:3  error  Unexpected empty test description  mocha/no-empty-description
```

Is that scenario also considered a bug?

**Comment by lo1tuma** [comment]
@neverendingqs do you get this message for the same code examples as mentioned above?

**Comment by neverendingqs** [comment]
> @neverendingqs do you get this message for the same code examples as mentioned above?

Yes.

**Comment by lo1tuma** [comment]
@neverendingqs I’ve deployed another fix in version 10.0.3. I hope I’ve covered all edge cases you described. Please let me know if you encounter any more issues with that rule.
